### ISSUE TITLE
Azure WordPress Multisite packaged by Bitnami - can't access site admin due to infinite redirects

### ISSUE THREAD

**Issue by sistemanick** [bitnami/vms#900]
### Platform

Azure

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

58a43d28-4bec-621f-7f65-08ab5d061495

### bndiagnostic was not useful. Could you please tell us why?

Basically just tells me I may have to increase "LimitInternalRecursion", which is not a good idea.

### Describe your issue as much as you can

After installing "WordPress Multisite packaged by Bitnami" on Azure, and set up two sites under the main network site, I'm unable to get to the dashboard of either of the two sites. I can browse the sites themselves though.

When I try to access the dashboard of either site, such as http://mysite/subsite/wp-admin, I get the "xxx redirected you too many times" error.

Another issue that might be relevant is that, when I try to use the DNS name of the VM to access the site (eg http://mysite.australiaeast.cloudapp.azure.com/), I get redirected to a URL like this http://xxx.xxx.xxx.xxx.nip.io/. where the xxx parts are just the IP of the VM. I don't have this issue with non-multisite Wordpress VMs.

**Comment by sistemanick** [comment]
Okay I've fixed this by following instructions here: https://docs.bitnami.com/aws/apps/wordpress-multisite/administration/use-directories/

But I'm still not sure why it's redirecting to http://xxx.xxx.xxx.xxx.nip.io/. I've tried:
1. Replacing the line in wp-config.php from `define( 'DOMAIN_CURRENT_SITE', 'xxx.xxx.xxx.xxx.nip.io' )` to `define( 'DOMAIN_CURRENT_SITE', 'mysite.australiaeast.cloudapp.azure.com' )`
2. Commented out the following block of code between `# BEGIN nip.io redirection` and `# END nip.io redirection` in both /apache/conf/vhosts/wordpress-vhost.conf and /apache/conf/vhosts/wordpress-https-vhost.conf.
